pages/index.js

```diff
@@ -1,6 +1,6 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import Head from "next/head";
+import Link from "next/link";
+import styles from "../styles/Home.module.css";
```

Comment:
We should talk about it, but I use to follow a convention about imports like the following:

```javascript
third-party libraries

local imports (components, types, etc)

styles imports

Example:
import Head from "next/head";

import MyLocalComponent from '../MyLocalComponent';

import styles from '../styles/index.css'
```

What do you think?

Reply:
Agree, that imports convention LGTM